src/test/java/api/support/fakes/FakeOkapi.java

```diff
@@ -560,12 +566,28 @@ private String findFakeCalendarById(String servicePointId, MultiMap queries) {
       String permanentLocation = holding.getString(ItemProperties.PERMANENT_LOCATION_ID);
       String temporaryLocation = holding.getString(ItemProperties.TEMPORARY_LOCATION_ID);
 
-      return item.put(ItemProperties.EFFECTIVE_LOCATION_ID,
+      return newItem.put(ItemProperties.EFFECTIVE_LOCATION_ID,
         ObjectUtils.firstNonNull(temporaryLocation, permanentLocation)
       );
     });
   }
 
+  private CompletableFuture<JsonObject> setItemStatusDateForItem(JsonObject oldItem,
+                                                                 JsonObject newItem) {
+    JsonObject oldItemStatus = oldItem.getJsonObject(ItemProperties.STATUS_PROPERTY);
+    JsonObject newItemStatus = newItem.getJsonObject(ItemProperties.STATUS_PROPERTY);
+    if(ObjectUtils.allNotNull(oldItemStatus, newItemStatus)){
+      if(!Objects.equals(oldItemStatus.getString("name"),
+        newItemStatus.getString("name"))){
+        DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+        df.setTimeZone(TimeZone.getTimeZone("UTC"));
+        String ISOCurrentDateTime = df.format(new Date());
+        newItemStatus.put("date", ISOCurrentDateTime);
```

Comment:
```suggestion
        newItemStatus.put("date", DateTime.now(DateTimeZone.UTC).toString());
```
I think we can replace all with the following code.

Reply:
I don`t think so. Because It should be like a real-life trigger with CURRENT_TIMESTAMP function, which returns DateTime as "2019-11-06T14:54:16.596+00:00", not "2019-11-06T14:54:16.590Z" (as in this case).

Reply:
ok, make sense.